### PR TITLE
non breaking UI enhancement

### PR DESCRIPTION
**Pull request by maelsar**
non-breaking UI enhancement just for cleaner (work-in-progress) temp UI
-merged section div for pair selection and connection
-separated div for orderbook
-applied grid styling with tailwind

-NOTE: remove the commented out previous code

### CODE REVIEW

File: src/app/OrderBook.tsx [hunk 1]
```diff
@@ -6,25 +6,59 @@ export function OrderBook({ orders }: { orders: OrderbookLine[] }) {
   //returns simple orderbook of buys/sells
   const adexState = useContext(AdexStateContext);
   const { buys, sells } = adexState.currentPairOrderbook;
+  // return (
```
Comment (EvgeniiaVak) [inline src/app/OrderBook.tsx:9]: I think we can go ahead and delete unused code, no need to keep it in comments - just delete it, putting it in comments just clutters everything.

File: src/app/OrderBook.tsx [hunk 2]
```diff
@@ -6,25 +6,59 @@ export function OrderBook({ orders }: { orders: OrderbookLine[] }) {
...
+  //   <>
+  //     {orders.map(
+  //       (
+  //         item: {
```
Comment (EvgeniiaVak) [inline src/app/OrderBook.tsx:13]: But also please note, that the work for OrderBook is in another branch: `order-book`, let's keep this branch about new Orders creation

**Review by EvgeniiaVak — CHANGES_REQUESTED** [review]
I think this one might cause a lot of conflicts with `order-book`. @maelsar I suggest we ship all the components separately (no styles, just functionality) and do the layout changes after they are ready.
For example, we could also add this element from the MVP feature list (I think noone is doing that one atm, and there is no issue about it yet, feel free to create one ):
Screenshot 2023-07-24 at 16 11 54
https://www.figma.com/file/P7pfzKwJ4G6ClapXfl61D3/Dexter-UI?type=design&node-id=352-30460&mode=design&t=ZEB91aKU18f3pQx3-0

**Comment by EvgeniiaVak** [comment]
I like the idea of tidying up the layout, but I suggest doing it after #6 is complete 
@maelsar thank you very much for your contribution, is it ok if we close this PR? Will you be able to apply these ideas to main branch after we are finished with #6 ?

**Comment by maelsar** [comment]
I agree completely to your comments and review.  Please go ahead and close the PR. Unfortunately, I am having some net problems due to weird weather just now. I still look forward to collaborating and contributing. Thanks!

**Pull request closed without merging**